database/Restaurant.js

```diff
@@ -57,4 +60,30 @@ const restaurantSchema = new mongoose.Schema({
 
 const Restaurant = mongoose.model('Restaurant', restaurantSchema);
 
+
+const find = (cb) => {
+  Restaurant.find().exec((err, docs) => {
+    if (err) {
+      cb(err);
+    } else {
+      console.log(docs);
```

Comment:
If this console.log is not being used for testing, I think it would look cleaner without this console.log.